include/zmq.h

```diff
@@ -683,6 +683,7 @@ ZMQ_EXPORT void zmq_threadclose (void *thread_);
 #define ZMQ_HELLO_MSG 110
 #define ZMQ_DISCONNECT_MSG 111
 #define ZMQ_PRIORITY 112
+#define ZMQ_BUSY_POLL 113
```

Comment:
this needs to be added to src/zmq_drafts.h as well, otherwise the non-draft build will fail

Reply:
Thank you for all reviews